src/bandwidth.rs

```diff
@@ -177,13 +148,13 @@ impl BandwidthSinks {
 
 /// Wraps around an `AsyncRead + AsyncWrite` and logs the bandwidth that goes through it.
 #[pin_project::pin_project]
-pub struct BandwidthConnecLogging<TInner> {
+pub struct BandwidthConnecLogging<SMInner> {
```

Comment:
```suggestion
pub struct InstrumentedStream<SMInner> {
```

This is no longer a connection, thus I would suggest a rename. How about `InstrumentedStream`?

Reply:
But that would be a breaking API change meaning we won't be able to release it in 0.50.1 and will have to wait 0.51, no?